src/status_im2/contexts/activity_center/notification/membership/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns status-im2.contexts.activity-center.notification.membership.view
+  (:require [i18n.i18n :as i18n]
```

Comment:
[utils.i18n :as i18n]

Reply:
Oh no, this one went through the cracks! Let me check if there are other bad usages in the activity center and fix them.